utils/lpac/utils.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <cjson/cJSON.h>
+#include <euicc/es9p.h>
+#include <euicc/es10b.h>
+#include <euicc/es10c.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define HTTP_ENV_NAME(DRIVER, NAME) "LPAC_HTTP_" #DRIVER "_" #NAME
```

Comment:
https://github.com/estkme-group/lpac/blob/abad55dd4d13b406e74834f215a4fc00e07d1124/src/main.c#L33-L34

move this a define to utils.h? share the string.

Reply:
No. It's only used by `src/main.c`, not generic.

Reply:
https://github.com/estkme-group/lpac/blob/abad55dd4d13b406e74834f215a4fc00e07d1124/driver/driver.c#L118-L119